models/_root/settings.yml

```diff
@@ -40,6 +40,25 @@ properties:
     description: Boolean that return true if bulk discount is enabled on products.
   cdn_url:
     type: string
+  channels:
+    type: object

```

Comment:
@bc-andreadao channels is an array of objects or does this type apply to one individual channel?

Reply:
Ah, @bc-yevhenii-buliuk  thanks for the catch! It's fixed now